tests/autoyast/installation.pm

```diff
@@ -55,8 +55,13 @@ sub run {
 
     push @needles, "autoyast-confirm"        if get_var("AUTOYAST_CONFIRM");
     push @needles, "autoyast-postpartscript" if get_var("USRSCR_DIALOG");
-    my $postpartscript = 0;
-    my $confirmed      = 0;
+    if (get_var("AUTOYAST_LICENSE")) {
+        push @needles, get_var("BETA") ? "inst-betawarning" : "autoyast-license";
```

Comment:
This should be caught here, as no "inst-betawarning" needles are searched in non-beta, but I can make it more obvious